src/content/configuration/output.md

```diff
@@ -498,13 +561,7 @@ output: {
 Module proof library.
 
 
-`libraryTarget: "assign"` - Here webpack will blindly generate an implied global.
-
-``` javascript
-MyLibrary = _entry_return_;
-```
-
-Be aware that if `MyLibrary` isn't defined earlier your library will be set in global scope.
+### Other libraryTargets
```

Comment:
I would simplify this title to just `Other Targets` the whole section is already in the context of `libraryTarget`. Also re your headers/sidebar question -- I think the other titles above should be simplified as well. This is a general problem we have across the site and while I do think we should try to keep titles more concise, we can also consider only showing `h2` level anchors in the sidebar.